src/UI/UIView.cpp

```diff
@@ -387,12 +434,45 @@ void UIView::draw_ui_callback()
 
           }
           else if (numVolumes > 0) {
+            if (m_compute_new_histogram) 
+            {
+             // std::vector<float> histogram = tfn_widget[m_table_selection].getHistogram();
```

Comment:
This is a lot of commented out code...

Reply:
I removed them. That code is never reached with the new Histogram implementation.